### PR TITLE
user-profile and logout

### PR DESCRIPTION
**Pull request by dejandeki123**
user profile and logout

### CODE REVIEW

**Comment by mjevtovic9** [comment]
There are conflicts on this PR

**Comment by dejandeki123** [comment]
solved conflicts

File: RentACar/src/app/user-profile/user-profile.component.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  constructor(private http: HttpClient,private userService: UserService,private router : Router) { }
+
+
+  ngOnInit(): void {
+    this.grabUser().subscribe(data => {this.user = data;})
+    
+  }
+
+  grabUser(): Observable<User> {
+   // this.userService.login(this.loginUser).subscribe(
```
Comment (mjevtovic9) [inline RentACar/src/app/user-profile/user-profile.component.ts:32]: remove commented code if it's not needed anymore

File: RentACar/src/app/user-profile/user-profile.component.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     // (data : any)=>{
+      let logovani = sessionStorage.getItem('token')
```
Comment (mjevtovic9) [inline RentACar/src/app/user-profile/user-profile.component.ts:34]: this variable is not used anywhere

File: RentACar/src/app/user-profile/user-profile.component.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+     //   sessionStorage.setItem('role', data.role)
+
+
+    return this.http.get<User>('http://localhost:8082/user/' + sessionStorage.getItem('username'));
```
Comment (mjevtovic9) [inline RentACar/src/app/user-profile/user-profile.component.ts:38]: this call should be in user.service.ts

**Pull request merged**